source/docs/wind/wind-toolkit/offshore-ca-download.html.md.erb

```diff
@@ -0,0 +1,276 @@
+---
+title: Offshore CA Data Download
+url: /api/wind-toolkit/v2/wind/offshore-ca-download
+---
+
+<p>The Offshore CA Dataset is a 20-year wind resource dataset for offshore California. Produced in 2020, this data set replaces NREL's Wind Integration National Dataset (WIND) Toolkit for offshore California, which was produced and released publicly in 2013 and is currently the principal data set used by stakeholders for wind resource assessment in the continental United States. Both the WIND Toolkit and this new data set are created using the Weather Research and Forecasting (WRF) numerical weather prediction model (NWP).</p>
```

Comment:
If you add these two paragraphs as summary and detail to the header [like so](https://github.com/NREL/developer.nrel.gov/pull/171/files#diff-317047b23b34d84f62bfd61fdb7c4e10L3-L4) they will show up in the right place on the previous page etc.

Reply:
Thanks @PjEdwards. I wasn't sure if the list should go in the detail frontmatter, or what the right format for it would be, but it doesn't look like detail shows up on the WTK index page.
![image](https://user-images.githubusercontent.com/502000/91999223-64c70d80-ecf9-11ea-9f10-fe8859000ddb.png)